client/components/credit-card/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CreditCard from '..';
+
+class CreditCards extends Component {
+	static displayName = 'CreditCard';
+
+	state = {
+		cards: [
+			{
+				card: '4242',
+				card_type: 'visa',
+				name: 'James Smith',
+				expiry: '05/23',
+			},
+			{
+				card: '4444',
+				card_type: 'mastercard',
+				name: 'Jean Davis',
+				expiry: '01/21',
+			},
+		],
+	};
+
+	render() {
+		return (
+			<Fragment>
+				{ this.state.cards.map( ( card, i ) => <CreditCard key={ i } card={ card } /> ) }
```

Comment:
the `key` here isn't unique to the card and this is one place where the bug can bite us - if we somehow remove a card from the list React could end up not redrawing or rearranging the others.

let's use something unique to the cards like their card number

```js
{ this.state.cards.map( card => <CreditCard key={ card.card } card={ card } /> ) }
```

Reply:
Done as part of https://github.com/Automattic/wp-calypso/pull/24485/commits/9c240b9f6bb3dee7cb127b730bc77b60b6fb1453